### PR TITLE
Homework v10 matsko

### PR DESCRIPTION
**Pull request by MykolaMatsko**
example code

### CODE REVIEW

File: homework.rb [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+  # [numbers.call, letters.call, numbers.call, letters.call].join("-")
+
+
+
+  letter = ("A".."Z").to_a
+
+  letters = lambda { |x| letter.sample(x).join }
+  number = lambda { rand(1..9).to_s }
+
+  number.call+number.call+number.call+"-"+letters.call(3)+"-"+number.call+number.call+number.call+"-"+letters.call(3)
```
Comment (sveredyuk) [inline homework.rb]: too long. try shorter

File: homework.rb [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
-end
+  # letter = ("A".."Z").to_a
+
+  # letters = -> { letter.sample(3).join }
+  # numbers = -> { rand(100..999).to_s }
+
+  # [numbers.call, letters.call, numbers.call, letters.call].join("-")
```
Comment (sveredyuk) [inline homework.rb]: don't commit commented code if it useless.

File: homework.rb [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
+
```
Comment (sveredyuk) [inline homework.rb]: useless spaces. Please, be neat

File: homework.rb (hunk 1 above)
Comment (sergiy-koshoviy) [inline homework.rb]: '+' - too expensive operator. Try to rewrite with interpolation

File: homework.rb [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+
+  letter = ("A".."Z").to_a
```
Comment (ruslan-369) [inline homework.rb]: `letter` - is not best name for array of letters)

**Comment by MykolaMatsko** [comment]
## I've created a new way of homework1. Including code refactoring and 100% probability sampling.

**Pull request closed without merging**